readme.md

```diff
@@ -1,151 +1,131 @@
+- Largest collection of languages and platforms (listed below)
+- Connects with private repositories and package registries
+
+### Languages
 
-- Get automated Pull Requests to update your dependencies
-- Reduce noise by running Renovate on a schedule, for example:
-  - on weekends
-  - outside of working hours
-  - each week
-  - each month
-- Relevant package files are discovered automatically
-- Supports monorepo architectures with workspaces with no extra configuration
-- Bot behavior is customizable via configuration files (config as code)
-- Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
-- Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
-- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions
-- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via the Mend Renovate App
+Renovate can provide updates for most popular languages, platforms, and registries including: npm, Java, Python, .NET, Scala, Ruby, Go, Docker and more.
+Supports over 90 different package managers.
 
-## Supported Platforms
+### Platforms
 
-Renovate works on these platforms:
+Renovate updates code repositories on the following platforms: GitHub, GitLab, Bitbucket, Azure DevOps, AWS Code Commit, Gitea, Forgejo, Gerrit (experimental)
 
-- [GitHub (.com and Enterprise Server)](https://docs.renovatebot.com/modules/platform/github/)
-- [GitLab (.com and CE/EE)](https://docs.renovatebot.com/modules/platform/gitlab/)
-- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
-- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
-- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
-- [AWS CodeCommit](https://docs.renovatebot.com/modules/platform/codecommit/)
-- [Gitea and Forgejo](https://docs.renovatebot.com/modules/platform/gitea/)
-- [Gerrit (experimental)](https://docs.renovatebot.com/modules/platform/gerrit/)
+## Ways to Run Renovate
 
-## Who Uses Renovate?
+The most effective way to run Renovate is to use an automated job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity.
+Mend offers cloud-hosted and self-hosted solutions. See the options below.
 
-Renovate is widely used in the developer community:
+### Mend Renovate Community (Cloud-Hosted)
 
-![Logos of companies and projects that use Renovate](./docs/usage/assets/images/matrix.png)
+**Supports: GitHub.com, Bitbucket Cloud**
 
-<details>
-<summary>List of companies and projects that use Renovate</summary>
+Hosted by Mend.io. No setup is required. Community plan available (Free)
```

Comment:
```suggestion
Hosted by Mend.io.
No setup is required.
Community plan available (Free).
```

Reply:
I am going for a shorter version (less page height), so prefer not to add additional line breaks.